app/serializers/disability_claim_detail_serializer.rb

```diff
@@ -34,6 +34,28 @@ def phase
     phase_from_keys 'claimPhaseDates', 'latestPhaseType'
   end
 
+  def tracked_items
+    items = []
+    items_src = object.data['consolidatedTrackedItemsList'] || []
```

Comment:
Looking at the requirements again, the front end should actually be able to construct all of this from the events timeline. The number of files needed will either be the count of items in the timeline with type `still_needed_from_you` or `still_needed_from_you + still_needed_from_others` (not sure if we are combining the two or not). The list of files received can be retrieved from the events timeline as well, using the types `received_from_you` and `received_from_others`